libs/core/src/global/styles/tokens/core/_core.scss

```diff
@@ -1,13 +1,21 @@
 /**
  * Do not edit directly
- * Generated on Fri, 12 Apr 2024 15:37:39 GMT
+ * Generated on Wed, 24 Apr 2024 16:03:41 GMT
  */
```

Comment:
Note for @anechol. We need to figure out how to prevent this from being added.

Reply:
Yes this is something I will look into.